ui/app/AppLayouts/Chat/controls/community/TokenItem.qml

```diff
@@ -0,0 +1,83 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.12
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+Control {
+    id: root
+
+    property string key
+    property string name
+    property string shortName
+    property url iconSource
+    property var subItems
+
+    signal itemClicked(string key, string name, string shortName,  url iconSource, var subItems)
+
+    leftPadding: 6 // by design
+    implicitHeight: 45 // by design
+    spacing: 8 // by design
+    background: Rectangle {
+        color: mouseArea.containsMouse ? Theme.palette.baseColor4 : "transparent"
+
+        MouseArea {
+            id: mouseArea
+            anchors.fill: parent
+            cursorShape: enabled ? Qt.PointingHandCursor : Qt.ArrowCursor
```

Comment:
could you please just root.enabled? just to make clear we're not using MouseArea's enabled